### ISSUE TITLE
Direct links for tested Raspbian OS

### ISSUE THREAD

**Issue by rohanrehman** [ryomuk/gu3000#3]
Could we have a direct link for:

Raspbery Pi Zero W
OS: Raspbian (Linux raspberrypi 5.4.79+ #1373)

kernel 5.10 has issues with the showfb example.
Testing on Pi Zero W.

**Comment by ryomuk** [comment]
Hi,
The Raspbian I used seems to be released on 2020-12-02, but now I couldn't find a link. I will try to fix the problem with kernel 5.10 or newer version.

**Comment by ryomuk** [comment]
It was a know issue.  See https://github.com/ryomuk/gu3000/issues/1
To fix:
in /boot/config.txt:
framebuffer_depth=16

I will update a document soon.

**Comment by rohanrehman** [comment]
Yes that was the fix I had to add and if using the 2020-12-02 release minimal, Overlay needs to be enabled in raspi-config.
